# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core/Cargo.toml
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-core/Cargo.toml
--- a/third_party/rust/wgpu-core/Cargo.toml	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-core/Cargo.toml	Tue Oct 07 17:00:40 2025 +0000
@@ -13,7 +13,7 @@
 edition = "2021"
 rust-version = "1.82.0"
 name = "wgpu-core"
-version = "26.0.0"
+version = "27.0.0"
 authors = ["gfx-rs developers"]
 build = "build.rs"
 autolib = false
@@ -144,7 +144,7 @@
 version = "0.4.21"
 
 [dependencies.naga]
-version = "26.0.0"
+version = "27.0.0"
 path = "../naga"
 
 [dependencies.once_cell]
@@ -189,11 +189,11 @@
 default-features = false
 
 [dependencies.wgpu-hal]
-version = "26.0.0"
+version = "27.0.0"
 path = "../wgpu-hal"
 
 [dependencies.wgpu-types]
-version = "26.0.0"
+version = "27.0.0"
 path = "../wgpu-types"
 default-features = false
 
@@ -201,12 +201,12 @@
 version = "0.2.1"
 
 [target.'cfg(all(target_arch = "wasm32", not(target_os = "emscripten")))'.dependencies.wgpu-core-deps-wasm]
-version = "26.0.0"
+version = "27.0.0"
 path = "platform-deps/wasm"
 optional = true
 
 [target.'cfg(any(windows, target_os = "linux", target_os = "android", target_os = "freebsd"))'.dependencies.wgpu-core-deps-windows-linux-android]
-version = "26.0.0"
+version = "27.0.0"
 path = "platform-deps/windows-linux-android"
 optional = true
 
@@ -215,12 +215,12 @@
 optional = true
 
 [target.'cfg(target_os = "emscripten")'.dependencies.wgpu-core-deps-emscripten]
-version = "26.0.0"
+version = "27.0.0"
 path = "platform-deps/emscripten"
 optional = true
 
 [target.'cfg(target_vendor = "apple")'.dependencies.wgpu-core-deps-apple]
-version = "26.0.0"
+version = "27.0.0"
 path = "platform-deps/apple"
 optional = true
 